python/ynab_client.py

```diff
@@ -1,15 +1,20 @@
 import settings
 
 from pynYNAB.Client import nYnabClient, nYnabConnection
-from pynYNAB.schema.Entity import Entity, ComplexEncoder, Base, AccountTypes
-from pynYNAB.schema.budget import Account, Transaction, Payee
-from pynYNAB.schema.roots import Budget
-from pynYNAB.schema.types import AmountType
+from pynYNAB.schema.budget import Transaction, Payee
 
-from sqlalchemy.sql.expression import select, exists, func
+from sqlalchemy.sql.expression import exists
+
+client = None
+connection = None
+
+def init():
+    global client
+    global connection
+
+    connection = connection or nYnabConnection(settings.ynab_username, settings.ynab_password)
```

Comment:
Perfect. So the next big thing is to allow sync on process boot somehow. We will eventually move this to happen in a background job, but that will always be optional (this needs to be super easy for people to install by default)